src/components/Header/Header.jsx

```diff
@@ -15,18 +17,19 @@ class Header extends Component {
     const { cookies } = props;
     this.toggleMenu = this.toggleMenu.bind(this);
     this.closeNewsBar = this.closeNewsBar.bind(this);
-    const { cookieValue } = 
-    this.state = { 
-      mobileMenu: false, 
-      visibleNewsBar: cookies.get('newsbar') ? false : true, 
-      sticky: false,
-      scrollDirection: 'bottom',
-      lastScrollPos: null
-    }
+    const { cookieValue } =
+      this.state = {
+        mobileMenu: false,
+        visibleNewsBar: cookies.get('newsbar') ? false : true,
+        visiblePrivacyPolicyBar: cookies.get('privacypolicybar') ? false : true,
```

Comment:
What about putting `newsbar` and `privacypolicybar` into a `const` instead of having them hard-coded?